examples/platform_view/lib/main.dart

```diff
@@ -47,6 +47,17 @@ class _MyHomePageState extends State<MyHomePage> {
     });
   }
 
+  Text _getButtonText() {
+    if (Platform.isAndroid) {
```

Comment:
A switch case statement might be more appropriate in this situation. IIRC, the linter will force you to have all options listed. That way it will be easier for macos team when its time to have a macos platform_view example.